scripts/tests/chiptest/test_definition.py

```diff
@@ -372,7 +372,8 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                     runner.RunSubprocess(python_cmd, name='CHIP_REPL_YAML_TESTER',
                                          dependencies=[apps_register], timeout_seconds=timeout_seconds)
             else:
-                pairing_cmd = paths.chip_tool_with_python_cmd + ['pairing', 'code', TEST_NODE_ID, setupCode]
+                pairing_cmd = paths.chip_tool_with_python_cmd + \
+                    ['pairing', 'code', TEST_NODE_ID, setupCode, 'skip-icd-registration', '0']
```

Comment:
Now that i think about this, should we be adding this?

Reply:
for manual test, we just provide the convenience to disable icd registration at default when using chip-tool based upon the ticket during certification test,
https://github.com/project-chip/connectedhomeip/issues/31992, However, we need exercise the ICD registration in auto-commissioning flow in CI, which is the real thing darwin/android platform would do in production.

Reply:
Does this affect all automated tests in CI? Test plan assumption is that chip-tool will not register by default.

Reply:
it only impacts the test cases when chip-tool validate with lit-example-app

Reply:
offline resolved.